client/app/components/_topbar/topbar.controller.js

```diff
@@ -24,6 +25,15 @@ class TopbarController {
     this.$mdSidenav("right").toggle();
   }
 
+  logOut(){
+    this.Authentication.logout()
+      .then(() => {
+        this.$location.hash("/login");
```

Comment:
Code style: we always used ui-router for state transitions. Is there a reason why you chose this instead of `$state.go("login")`?